app/models/health/health_file.rb

```diff
@@ -60,11 +60,13 @@ def set_calculated!(user_id, client_id)
       self.user_id ||= user_id
       self.client_id ||= client_id
 
-      if File.exist?(file.path)
-        self.content ||= file.read
-        self.content_type ||= file.content_type
-        self.size ||= content&.size
-        self.name ||= file.filename
+      if File.exist?(file.path) && file.read.present?
```

Comment:
I think we'd want to store `file.read` in a variable and check that.  Otherwise you'll need `file.rewind` before we read it again.

Reply:
Also, maybe if `self.content.present?` we don't want to do any of this?